pkg/ebpf/verifier/calculator/main.go

```diff
@@ -53,6 +70,18 @@ func main() {
 			return nil
 		}
 
+		if len(filterFiles) > 0 {
+			found := false
+			for _, f := range filterFiles {
+				if strings.TrimSuffix(d.Name(), ".o") == f {
```

Comment:
The current way would match a non-`.o` file if it matched the filter. If you intend to only match `.o` files, invert the match:
```suggestion
				if d.Name() == fmt.Sprintf("%s.o", f) {
```

Reply:
Or put the `.o` filter before this loop.

Reply:
Removing any operation on the filter. Will use it as the user provided.